### PR TITLE
fix: icon style

### PR DESCRIPTION
**Pull request by Randolph314**
## Description

Closes  [MASK-667](https://mask.atlassian.net/browse/MASK-667)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
  - [x] I have removed all in development `console.log`s
  - [x] I have removed all commented code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Review by UncleBill — DISMISSED** [review]

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1197,6 +1197,21 @@ packages:
       web3: 0.20.7
     dev: false
 
+  /@0xproject/abi-gen/0.1.2:
+    resolution: {integrity: sha512-vwArB0lTTuInOWTh5KeeNeJTswrGJrHebau8sdrG8TBNiVwAAehthfWflieMRKK54D11Z+7TO8fupCMa7BaoxA==}
```
Comment (guanbinrui) [inline pnpm-lock.yaml:1201]: Do not change this file without touch package.json.

**Review by Jack-Works — DISMISSED** [review]

**Review by septs — CHANGES_REQUESTED** [review]
plz resolve conflict

**Review by septs — CHANGES_REQUESTED** [review]
plz use imageoptim reduce image resources

**Comment by Jack-Works** [comment]
There is a merge conflict marker committed. Please revert that change.

**Review by Jack-Works — DISMISSED** [review]

**Comment by wenluomask** [comment]
Checked.

**Review by yanzhihong23 — APPROVED** [review]

**Review by Lanttcat — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**